pkg/eventprocessor/impl/webhook.go

```diff
@@ -76,14 +80,162 @@ func sendWebhookRequest(ctx context.Context, url string, body interface{}) error
+		BlockExplorerURL: "https://nova.arbiscan.io",
+		SupportsNFTView:  false,
+	},
+	314: {
+		ID:               314,
+		Name:             "Filecoin",
+		ContractAddr:     common.HexToAddress("0x59EF8Bf2d6c102B4c42AEf9189e1a9F0ABfD652d"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/Filecoin",
+		BlockExplorerURL: "https://filfox.info/en",
+		SupportsNFTView:  false,
+	},
+	5: {
+		ID:               5,
+		Name:             "Ethereum Goerli",
+		ContractAddr:     common.HexToAddress("0xDA8EA22d092307874f30A1F277D1388dca0BA97a"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/ethereum",
+		BlockExplorerURL: "https://goerli.etherscan.io",
+		SupportsNFTView:  true,
+	},
+	11155111: {
+		ID:               11155111,
+		Name:             "Ethereum Sepolia",
+		ContractAddr:     common.HexToAddress("0xc50C62498448ACc8dBdE43DA77f8D5D2E2c7597D"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/ethereum",
+		BlockExplorerURL: "https://sepolia.etherscan.io",
+		SupportsNFTView:  true,
+	},
+	420: {
+		ID:               420,
+		Name:             "Optimism Goerli",
+		ContractAddr:     common.HexToAddress("0xC72E8a7Be04f2469f8C2dB3F1BdF69A7D516aBbA"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/optimism",
+		BlockExplorerURL: "https://blockscout.com/optimism/goerli",
+		SupportsNFTView:  false,
+	},
+	421613: {
+		ID:               421613,
+		Name:             "Arbitrum Goerli",
+		ContractAddr:     common.HexToAddress("0x033f69e8d119205089Ab15D340F5b797732f646b"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/arbitrum",
+		BlockExplorerURL: "https://goerli-rollup-explorer.arbitrum.io",
+		SupportsNFTView:  false,
+	},
+	314159: {
+		ID:               314159,
+		Name:             "Filecoin Calibration",
+		ContractAddr:     common.HexToAddress("0x030BCf3D50cad04c2e57391B12740982A9308621"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/Filecoin",
+		BlockExplorerURL: "https://calibration.filfox.info/en",
+		SupportsNFTView:  false,
+	},
+	80001: {
+		ID:               80001,
+		Name:             "Polygon Mumbai",
+		ContractAddr:     common.HexToAddress("0x4b48841d4b32C4650E4ABc117A03FE8B51f38F68"),
+		TBLDocsURL:       "https://docs.tableland.xyz/fundamentals/chains/polygon",
+		BlockExplorerURL: "https://mumbai.polygonscan.com",
+		SupportsNFTView:  false,
+	},
+}
+
+type webhookContentData struct {
+	ChainID       tableland.ChainID
+	TBLDocsURL    string
+	BlockNumber   int64
+	TxnHash       string
+	TxnURL        string
+	TableIDs      string
+	Error         *string
+	ErrorEventIdx *int
+}
+
+// getTableNFTViews returns the NFT views for the given table IDs.
+func getTableNFTViews(tableIDs tables.TableIDs, chainID tableland.ChainID) string {
+	var tableNFTURLs []string
+	for _, tableID := range tableIDs {
+		if !chains[int64(chainID)].SupportsNFTView {
```

Comment:
worth creating a variable for storing the value of `chains[int64(chainID)]`, so you don't do lots map lookup later? mostly to increase readability